scripts/sysman/memory.yml

```diff
@@ -132,6 +132,21 @@ members:
       desc: "[out] The total allocatable memory in bytes (can be less than the `physicalSize` member of $s_mem_properties_t)"
 --- #--------------------------------------------------------------------------
 type: struct
+desc: "Extension properties for Memory State"
+version: "1.7"
+class: $sMemory
+name: $s_mem_ext_state_t
+details:
+    - "Memory Page Offline Metrics"
+members:
+    - type: uint32_t
+      name: memoryPageOffline
+      desc: "[out] Returns the number of Memory Pages Offline"
+    - type: uint32_t
+      name: maxMemoryPageOffline
```

Comment:
Please share how this could be used by the Application.

Reply:
Updated the structure with exp. Also added the new files for adding extension.